README.rst

```diff
@@ -138,6 +138,20 @@ To receive results in ``pandas`` format, use the ``get_dataframe()`` method:
 
 You can specify any of the end of day frequencies (daily, weekly, monthly, and annually) or any intraday frequency for both the ``get_ticker_price`` and ``get_dataframe`` methods.  Weekly frequencies resample to the end of day on Friday, monthly frequencies resample to the last day of the month, and annually frequencies resample to the end of day on 12-31 of each year.  The intraday frequencies are specified using an integer followed by "Min" or "Hour", for example "30Min" or "1Hour".
 
+
+You can obtain top-of-book Cryptocurrency quotes from the ``get_crypto_top_of_book()`` method.
```

Comment:
Cryptocurrency should be lowercase, as it's not a proper noun.

Reply:
Hey @hydrosquall, I'm not sure if the changes I made are showing for you. I did make the requested changes, but I see no record of them. I might be lost in the GitHub ether here.

Reply:
Hi @GenusGeoff, thanks for checking in. Currently I only see the changes from December (screenshot [here](
https://a.cl.ly/nOulryXm)). 

Is it possible that the changes are on your local git repository, but haven't been pushed up to Github yet? 

Alternately, if you have the raw text for what you intended to add, feel free to post it as a comment on this PR, and I can help you get it added to the branch.